File: sdk/dart/lib/model/line_item.dart
part of swagger.api;


@Entity()
class LineItem {
  
  @Property(name: 'ID')
  String ID = null;
  

  @Property(name: 'ProductID')
  String productID = null;
  

  @Property(name: 'Quantity')
  int quantity = null;
  

  @Property(name: 'UnitPrice')
  Number unitPrice = null;
  

  @Property(name: 'CostCenter')
  String costCenter = null;
  

  @Property(name: 'DateNeeded')
  DateTime dateNeeded = null;
  

  @Property(name: 'ShippingAccount')
  String shippingAccount = null;
  

  @Property(name: 'ShippingAddressID')
  String shippingAddressID = null;
  

  @Property(name: 'ShipfromAddressID')
  String shipfromAddressID = null;
  

  @Property(name: 'ShipperID')
  String shipperID = null;
  

  @Property(name: 'Specs')
  List<ShipmentItem> specs = [];
  

  @Property(name: 'xp')
  Object xp = null;
  
  LineItem();

  @override
  String toString()  {
    return 'LineItem[ID=$ID, productID=$productID, quantity=$quantity, unitPrice=$unitPrice, costCenter=$costCenter, dateNeeded=$dateNeeded, shippingAccount=$shippingAccount, shippingAddressID=$shippingAddressID, shipfromAddressID=$shipfromAddressID, shipperID=$shipperID, specs=$specs, xp=$xp, ]';
  }

}

File: sdk/javascript-closure-angular/API/Client/LineItem.js
goog.provide('API.Client.LineItem');

/**
 * @record
 */
API.Client.LineItem = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.LineItem.prototype.ID;

/**
 * @type {!string}
 * @export
 */
API.Client.LineItem.prototype.productID;

/**
 * @type {!number}
 * @export
 */
API.Client.LineItem.prototype.quantity;

/**
 * @type {!number}
 * @export
 */
API.Client.LineItem.prototype.unitPrice;

/**
 * @type {!string}
 * @export
 */
API.Client.LineItem.prototype.costCenter;

/**
 * @type {!API.Client.date}
 * @export
 */
API.Client.LineItem.prototype.dateNeeded;

/**
 * @type {!string}
 * @export
 */
API.Client.LineItem.prototype.shippingAccount;

/**
 * @type {!string}
 * @export
 */
API.Client.LineItem.prototype.shippingAddressID;

/**
 * @type {!string}
 * @export
 */
API.Client.LineItem.prototype.shipfromAddressID;

/**
 * @type {!string}
 * @export
 */
API.Client.LineItem.prototype.shipperID;

/**
 * @type {!Array<!API.Client.ShipmentItem>}
 * @export
 */
API.Client.LineItem.prototype.specs;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.LineItem.prototype.xp;

